### PR TITLE
Added Single video Page

### PR DESCRIPTION
**Pull request by saurabhchirde**
Features added

1. User Can view Video on fullscreen
2. Can place like while watching the video.
3. Can Add to Watch later from the video page.

single-video

### CODE REVIEW

**Review by m26unkwn — APPROVED** [review]
Could not find any issue.

Good to go ahead

File: src/Pages/SingleVideo/SingleVideo.jsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { VideoPlayer } from "../../Components/VIdeoPlayer/VideoPlayer";
+import { useAuth, useAxiosCalls, useModal, useVideo } from "../../Context";
+import "./SingleVideo.css";
+import { VerticalCard } from "../../Components/Cards/VerticalCard";
```
Comment (mehulsatardekar) [inline src/Pages/SingleVideo/SingleVideo.jsx:6]: Just a suggestion, you can export all the components in one (index.js) file and import it from there

**Review by mehulsatardekar — APPROVED** [review]
Rest looks good from my side, love that you have followed good file structure and naming convention.

File: src/Pages/SingleVideo/SingleVideo.jsx (hunk 1 above)
Comment (saurabhchirde) [reply src/Pages/SingleVideo/SingleVideo.jsx:6]: Ok, will fix that.

**Pull request merged**